src/benchpress/evaluation/engine.py

```
@@ -145,103 +163,181 @@ async def evaluate_task(
-                    model_output = await self.model.generate(prompt)
-                    # Format the model output for display using the new unicodeit-based formatter
-                    formatted_output = latex_to_unicode(model_output)
-                    self.console.print(
-                        Panel(
-                            formatted_output,
-                            title="Model Response",
+                    # Use streaming if enabled
+                    if self.streaming:
+                        # Stop the progress bar before streaming
+                        progress.stop()
+                        
```

Ruff (0.8.2)

Blank line contains whitespace

Remove whitespace from blank line

(W293)